.github/workflows/build-test

```diff
@@ -51,6 +51,8 @@ cd ${GITHUB_WORKSPACE}/tests
 
 python -m pip install --pre -r test-requirements.txt
 
+python -m pip install --upgrade --pre pytket~=1.0
```

Comment:
Perhaps a comment here would be good, just to explain that we want to test against pre-release versions.

Reply:
done